libs/indibase/indiccd.cpp

```diff
@@ -2579,7 +2594,8 @@ bool CCD::uploadFile(CCDChip * targetChip, const void * fitsData, size_t totalBy
             prefix = std::regex_replace(prefix, std::regex("XXX"), prefixIndex);
         }
 
-        snprintf(imageFileName, MAXRBUF, "%s/%s%s", UploadSettingsT[0].text, prefix.c_str(), targetChip->FitsB.format);
+        // snprintf(imageFileName, MAXRBUF, "%s/%s%s", UploadSettingsTP[UPLOAD_DIR].getText(), prefix.c_str(), targetChip->FitsBP[0].getFormat());
+        strcpy(imageFileName, targetChip->FitsBP[0].getFormat());
```

Comment:
I'm just using this now. But not able to set imageFileName since it is char imageFileName[MAXRBUF];
  
std::string string = UploadSettingsTP[UPLOAD_DIR].getText() + std::string(targetChip->FitsBP[0].getFormat());

Reply:
std::string imageFileName = UploadSettingsTP[UPLOAD_DIR].getText() + "/" + prefix + "/" + std::string(FitsBP[0].getFormat());